platform/clickhouse/schema.go

```diff
@@ -220,35 +221,75 @@ func NewBaseType(clickHouseTypeName string) BaseType {
 // this is catch all type for all types we do not exlicitly support
 type UnknownType struct{}
 
-func ResolveType(clickHouseTypeName string) reflect.Type {
-	switch clickHouseTypeName {
-	case "String", "LowCardinality(String)", "UUID", "FixedString":
+func ResolveType(dorisTypeName string) reflect.Type {
```

Comment:
I think we have to have type resolver per each backend. It would make sense to introduce 
```
type TypeResolver interface {
  ResolveType(typeName string) reflect.Type
}
```